src/node/plugin-mdx/pluginMdxGit.ts

```diff
@@ -0,0 +1,36 @@
+import { Plugin } from 'vite';
+import { simpleGit } from 'simple-git';
+import { MD_REGEX } from '../../node/constants';
+
+export function pluginMdxGit(): Plugin {
+  const cache = new Map<string, string>();
+  const git = simpleGit();
+
+  // https://github.com/steveukx/git-js#git-log
+  async function getLastUpdatedTime(path: string) {
+    const { latest } = await git.log({ file: path });
+    return !latest ? '' : latest.date;
+  }
+
+  return <Plugin>{
+    name: 'vite-plugin-mdx-git',
+    async transform(code, id) {
```

Comment:
Maybe we should clean the url without query and hash there because the hmr request always take `?import&t=xxx` params.

Reply:
Sorry I didn't get it. I've tried to edit the md(x) content, and the id remained same